GSA/digitalgov.gov
# Added hostname to og:url to include top level domain

## Summary
Adds top level host name to `og:url` meta tags. Closes #7168.

### Preview
[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-fix-og-url/)

### Solution

When viewing the `og:url` value on the live site, only the file pathname (`/communities/`) is displayed instead of `https://www.digital.gov/communities/`.

By setting a `hostName` params in the `config.yml` we can set the base host name.

> [!NOTE]
> `og:url` path will be different for each environment

**production**
`{{ .Permalink }}` will only return the file path and not the host name:
`https://digital.gov/resources/an-introduction-to-domain-management`

**cloud pages preview**
`{{ .Permalink }}` returns the preview path and branch name with the file path:
`https://digital.gov/preview/gsa/digitalgov.gov/nl-fix-og-url/resources/an-introduction-to-domain-management`

**localhost**
`{{ .Permalink }}` will return localhost:1313 with the file path:
`https://digital.gov/localhost:1313/resources/an-introduction-to-domain-management`

### How To Test

1. Visit any page
2. Open developer tools and search for `og:url`
3. Should see `https://digital.gov/preview/gsa/digitalgov.gov/nl-fix-og-url/resources/an-introduction-to-domain-management`
4. In production this will resolve to `https://digital.gov/resources/an-introduction-to-domain-management`

---

### Dev Checklist
